configuration_helper/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2014 David BEAL Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Configuration Helper",
+    "version": "16.0.1.0.0",
+    "author": "Akretion,Odoo Community Association (OCA)",
+    "maintainer": "Akretion",
+    "category": "server",
+    "depends": ["base", "base_setup"],
```

Comment:
```suggestion
    "depends": ["base"],
```

Reply:
we can also keep the `company_id` in the field configuration_helper/models/config.py

Reply:
the test worked on my local, ty 🙏

Reply:
> it is not work on my local if I don't depend base_setup because company_id is defined in this module

Reply:
if so, a fresh install reported this issue

```
  File "/Users/tran/code/odoo/odoo/16.0/odoo/fields.py", line 536, in setup
    self.setup_related(model)
  File "/Users/tran/code/odoo/odoo/16.0/odoo/fields.py", line 602, in setup_related
    f"Field {name} referenced in related field definition {self} does not exist."
KeyError: 'Field company_id referenced in related field definition abstract.config.settings.something does not exist.'
```